# Module 69.1 Study - Forwards and Futures

<hr>

### LOS 69.a: Define forward contracts, futures contracts, swaps, options (calls and puts), and credit derivatives and compare their basic characteristics.

#### Forward Contracts

In a **forward contract** between two parties, one party (the buyer) commits to buy and the other party (the seller) commits to sell a physical or financial asset at a specific price on a specific date (the settlement date) in the future.

The buyer has long exposure to the underlying asset in that he will make a profit on the forward if the price of the underlying at the settlement date exceeds the forward price, and have a loss if the price of the underlying at the settlement date is less than the forward price. The results are opposite for the seller of the forward, who has short exposure to the underlying asset.

#### Futures Contracts

A **futures contract** is quite similar to a forward contract but is standardized and exchange-traded. The primary ways in which forwards and futures differ are that futures trade in a liquid secondary market, are subject to greater regulation, and trade in markets with more disclosure (transparency). Futures are backed by a central clearinghouse and require daily cash settlement of gains and losses, so that counterparty credit risk is minimized.

On a futures exchange, **margin** is cash or other acceptable collaterals that both the buyer and seller must deposit. Unlike margin in bond or stock accounts, there is no loan involved and, consequently, there are no interest charges. This collateral provides protection for the clearinghouse. At the end of each trading day, the margin balance in a futures account is adjusted for any gains and losses in the value of the futures position based on the new settlement price, a process called the **mark-to-market** or marking-to-market. The settlement price is calculated as the average price of trades over a period at the end of the trading session.

**Initial margin** is the amount of cash or collateral that must be deposited in a futures account before a trade may be made. Initial margin per contract is relatively low and is approximately one day's maximum expected price fluctuation on the total value of the assets covered by the contract.

**Maintenance margin** is the minimum amount of margin that must be maintained in a futures account. If the margin balance in the account falls below the maintenance margin through daily mark to market from changes in the futures price, the account holder must deposit additional funds to bring the margin balance back up to the initial margin amount, or the exchange will close out the futures position. This is different from a margin call in an equity account, which requires investors only to bring the margin back up to the maintenance margin amount. Futures margin requirements are set by the exchange.

To illustrate the daily mark-to-market for futures, consider a contract for 100 ounces of gold that settles on May 15. The initial margin amount is £5,000 and the maintenance margin is £4,700.

On Day 0

* A buyer and seller make a trade at the end of the day at a price of £1,950 per ounce and both parties deposit the initial margin of £5,000 into their accounts.

On Day 1, the settlement price falls to £1,947.50. The seller has gains and the buyer has losses.

* The exchange will credit the seller's account for (1,950 – 1,947.50) × 100 = £250, increasing the margin balance to £5,250.
* The exchange will deduct (1,950 – 1,947.50) × 100 = £250 from the buyer's account, decreasing the margin balance to £4,750. Because £4,750 is more than the maintenance (minimum) margin amount of £4,700, no additional deposit is required.


On Day 2, the settlement price falls to £1,945. Again, the seller has gains and the buyer has losses.

* The exchange will credit the seller's account for (1,947.50 – 1,945) × 100 = £250, increasing their margin balance to £5,500.
* The exchange will deduct (1,947.50 – 1,945) × 100 = £250 from the buyer's account, decreasing the margin balance to £4,500. Because £4,500 is less than the maintenance (minimum) margin amount of £4,700, the buyer must deposit 5,000 – 4,500 = £500 into their margin account to return it to the initial margin amount of £5,000.
* At the end of Day 2, both parties have futures positions at the new settlement price of £1,945 per ounce.

Many futures contracts have **price limits**, which are exchange-imposed limits on how much each day's settlement price can change from the previous day's settlement price. Exchange members are prohibited from executing trades at prices outside these limits. If the equilibrium price at which traders would willingly trade is above the upper limit or below the lower limit, trades cannot take place. Some exchanges have **circuit breakers**; in this case, when a futures price reaches a limit price, trading is suspended for a short period.

<hr>

# Module 69.2: Swaps and Options

<hr>

#### Swaps

Swaps are agreements to exchange a series of payments on multiple settlement dates over a specified time period (e.g., quarterly payments for two years). At each settlement date, the two payments are netted so that only one net payment is made. The party with the greater liability at each settlement date pays the net difference to the other party.

Swaps trade in a dealer market and the parties are exposed to counterparty credit risk, unless the market has a central counterparty structure to reduce counterparty risk. In this case, margin deposits and mark-to-market payments may also be required to further reduce counterparty risk.

We can illustrate the basics of a swap with a simple fixed-for-floating interest rate swap for two years with quarterly interest payments based on a **notional principal amount** of £10 million. In such a swap, one party makes quarterly payments at a fixed rate of interest (the **swap rate**) and the other makes quarterly payments based on a floating **market reference rate**.

The swap rate is set so that the swap has zero value to each party at its inception. As expectations of future values of the market reference rate change over time, the value of the swap can become positive for one party and negative for the other party.

Consider an interest rate swap with a notional principal amount of £10 million, a fixed rate of 2%, and a floating rate of the 90-day secured overnight financing rate (**SOFR**). At each settlement date, the fixed-rate payment will be `£10 million × 0.02/4 = £50,000`. The floating-rate payment at the end of the first quarter will be based on 90-day SOFR at the initiation of the swap, so that both payments are known at the inception of the swap.

If, at the end of the first quarter, 90-day SOFR is 1.6%, the floating-rate payment at the second quarterly settlement date will be `£10 million × 0.016 / 4 = £40,000`. The fixed-rate payment is again £50,000, so at the end of the second quarter the fixed-rate payer will pay the net amount of £10,000 to the other party.

A company with 2-year floating-rate quarterly-pay note outstanding could enter such a swap as the fixed-rate payer, converting its floating-rate liability into a fixed-rate liability. It now makes fixed interest rate payments and can use the floating-rate payments from the counterparty to make the payments on its floating-rate debt. By entering into the swap, the company can hedge the interest rate risk (uncertainty about future quarterly rates) of their existing floating-rate liability.

As we will see in our reading on swap valuation, a swap can be constructed from a series of forward contracts in which the underlying is a floating rate and the forward price is a fixed rate. Each forward settles on one of the settlement dates of the swap. At each settlement date, the difference between the fixed and the floating rates would result in a net payment, just as with a swap. Often, interest rate forwards settle at the beginning of the quarter rather than the end; the cash flows are the present value equivalents of the end-of-quarter swap payments.

#### Credit Swaps

One type of swap that is structured a bit differently is a **credit default swap (CDS)**. With a CDS, the protection buyer makes fixed payments on the settlement dates and the protection seller pays only if the underlying (a reference security) has a **credit event**. This could be a bond default, a corporate bankruptcy, or an involuntary restructuring.

When a credit event occurs, the protection seller must pay an amount that offsets the loss in value of the reference security. The fixed payments represent the yield premium on the reference bonds that compensates bondholders for the expected loss from default, the probability of default times the expected loss in the event of default (or other credit event). The protection buyer is essentially paying the yield premium on the reference security for insurance against default.

The holder of a risky bond can hedge its default risk by entering a CDS as the protection buyer. The protection seller receives the default risk premium (<u>credit spread</u>) and takes on the risk of default, resulting in risk exposure similar to that of holding the reference bond.

#### Options

The two types of options of interest to us here are put options and call options on an underlying asset. We introduce them using option contracts for 100 shares of a stock as the underlying asset.

A put option gives the buyer the right (but not the obligation) to sell 100 shares at a specified price (the exercise price, also referred to as the strike price) for specified period of time, the time to expiration. The put seller (also called the writer of the option) takes on the obligation to purchase the 100 shares at the price specified in the option, if the put buyer exercises the option.

Note the "one-way" nature of options. If the exercise price of the puts is £25 at the expiration of the option, and the shares are trading at or above £25, the put holder will not exercise the option. There is no reason to exercise the put and sell shares at £25 when they can be sold for more than £25 in the market. This is the outcome for any stock price greater than or equal to £25. Regardless of whether the stock price at option expiration is £25 or £1,000, the put buyer lets the option expire, and the put seller keeps the proceeds from the sale.

If the stock price is below £25, the put buyer will exercise the option and the put seller must purchase 100 shares for £25 from the put buyer. On net, the put buyer essentially receives the difference between the stock price at expiration and £25 (times 100 shares).

A call option gives the buyer the right (but not the obligation) to buy 100 shares at a specified price (the exercise price) for a specified period of time. The call seller (writer) takes on the obligation to sell the 100 shares at the exercise price, if the call buyer exercises the option.

<hr> 

### LOS 69.b: Determine the value at expiration and profit from a long or a short position in a call or put option.

Unlike forwards, futures, and swaps, options are sold at a price (they do not have zero value at initiation). The price of an option is also referred to as the option premium.

At expiration the payoff (value) of a call option to the owner is Max $(0, S – X)$, where S is the price of the underlying at expiration and $X$ is the exercise price of the call option. The Max ( ) function tells us that if $S < X$ at expiration, the option value is zero, that is, it expires worthless and will not be exercised.

At expiration the payoff (value) of a put option to the owner is Max $(0, X – S)$, where $S$ is the price of the underlying at expiration and X is the exercise price of the put option. A put has a zero value at expiration unless $X – S$ is positive.

For the buyer of a put or call option, the profit at expiration is simply the difference between the value (payoff) of the option at expiration and the premium the investor paid for the option.

Because the seller (writer) of an option receives the option premium, the profit to the option seller at expiration is the amount of the premium received minus the option payoff at expiration. The writer loses the payoff at expiration and will have a loss on the option if the payoff is greater than the premium received.

Note the risk exposures of call and put buyers and writers. The buyer of a put or call has no further obligation, so the maximum loss to the buyer is simply the amount they paid for the option. The writer of a call option has exposure to an unlimited loss because the maximum price of the underlying, $S$, is (theoretically) unlimited, so that the payoff $S – X$ is unlimited. The payoff on a put option is $X – S$, so if the lower limit on $S$ is zero, the maximum payoff on a put option is the exercise price, $X$.

#### Call Option Profits and Losses

Consider a call option with a premium of £5 and an exercise price of £50. This means the buyer pays £5 to the writer. At expiration, if the price of the stock is less than or equal to the £50 exercise price, the option has zero value, the buyer of the option is out £5, and the writer of the option is ahead £5. When the stock's price exceeds £50, the option starts to gain (breakeven will come at £55, when the value of the stock equals the strike price plus the option premium). Conversely, as the price of the stock moves upward, the seller of the option starts to lose (negative figures will start at £55, when the value of the stock equals the exercise price and the option premium).

An illustration of the profit or loss at expiration for the buyer (long) and writer (short) of this call option, as a function of the stock price, is presented in  **Profit/Loss Diagram** for a Call Option. This profit/loss diagram indicates the following:


* The maximum loss for the buyer of a call is the £5 premium (at any S ≤ £50).
* The breakeven point for the buyer and seller is the exercise price plus the premium (at S = £55).
* The profit potential to the buyer of the option is unlimited, and, conversely, the potential loss to the writer of the call option is unlimited.
* The call holder will exercise the option whenever the stock's price exceeds the exercise price at the expiration date.
* The greatest profit the writer can make is the £5 premium (at any S ≤ £50).
* The sum of the profits between the buyer and seller of the call option is always zero; thus, trading options is a zero-sum game. One party's profits equal the other party's losses.


*Profit/Loss Diagram for a Call Option*

<IMG SRC="https://github.com/PachaTech/CFA-Level-1/blob/main/Derivatives/Module%2069/pics/1-17-78-1.jpg?raw=true"></img>



#### Put Option Profits and Losses


To examine the profits and losses associated with trading put options, consider a put option with a £5 premium and a £50 exercise price. The buyer pays £5 to the writer. When the price of the stock at expiration is greater than or equal to the £50 exercise price, the put has zero value. The buyer of the option has a loss of £5 and the writer of the option has a gain of £5. When the stock price is less than £50, the put option has a positive payoff. Breakeven will come at £45, when the value of the stock equals the exercise price less the option premium. At a stock price below £45, the put seller will have a loss.

**Profit/Loss Diagram for a Put Option** shows the profit/loss diagram for the buyer (long) and seller (short) of the put option that we have been discussing. This profit/loss diagram illustrates that:

* The maximum loss for the buyer of a put is the £5 premium (at any S ≥ £50).
* The maximum gain to the buyer of a put is limited to the exercise price less the premium (£50 – £5 = £45). The potential loss to the writer of the put is the same amount.
* The breakeven price of a put buyer (seller) is the exercise price minus the option premium (£50 – £5 = £45).
* The maximum profit for the writer is the £5 premium (S ≥ £50).
* The profit (loss) of the put buyer will always equal the loss (profit) of the put writer.


*Profit/Loss Diagram for a Put Option*

<IMG SRC="https://github.com/PachaTech/CFA-Level-1/blob/main/Derivatives/Module%2069/pics/put.jpeg?raw=true"></img>


#### Example: Option profit calculations

Suppose that both a call option and a put option have been written on a stock with an exercise price of £40. The current stock price is £42, and the call and put premiums are £3 and £0.75, respectively.

Calculate the profit to the long and short positions for both the put and the call with an expiration day stock price of £35 and with a price at expiration of 43.

In [9]:
### Payoff of various options

### Stock trading at 35
long_call = 0 - 3 
print("long_call @ 35: ",long_call)
short_call = 3 - 0 
print("short_call @ 35: ",(short_call))
long_put = 5 - 0.75
print("Long_put @ 35: ",(long_put))
short_put = 0.75 - 5
print("short_putl @ 35: ",(short_put))

# Print break
print("")
print("")

### Stock trading at 43
long_call2 = -3 + 3 
print("long_call @ 43: ",long_call2)
short_call2 = 3 - 3 
print("short_call @ 43: ",(short_call2))
long_put2 = 0.75 - 0
print("Long_put @ 43: ",(long_put2))
short_put2 = 0.75 - 0
print("short_putl @ 43: ",(short_put2))

long_call @ 35:  -3
short_call @ 35:  3
Long_put @ 35:  4.25
short_putl @ 35:  -4.25


long_call @ 43:  0
short_call @ 43:  0
Long_put @ 43:  0.75
short_putl @ 43:  0.75


#### Answer:

Profit will be computed as ending option value – initial option cost.

Stock at £35:

* Long call: £0 – £3 = –£3. The option has no value, so the buyer loses the premium paid.
* Short call: £3 – £0 = £3. Because the option has no value, the call writer's gain equals the premium received.
* Long put: £5 – £0.75 = £4.25. The buyer paid £0.75 for an option that is now worth £5.
* Short put: £0.75 – £5 = –£4.25. The seller received £0.75 for writing the option, but the option will be exercised so the seller will lose £5 at expiration.

Stock at £43:

* Long call: –£3 + £3 = £0. The buyer paid £3 for the option, and it is now in the money by $3. Hence, the net profit is zero.
* Short call: £3 – £3 = £0. The seller received £3 for writing the option and now faces a –£3 valuation because the buyer will exercise the option, for a net profit of zero.
* Long put: –£0.75 – £0 = –£0.75. The buyer paid £0.75 for the put option and the option is now worthless.
* Short put: £0.75 – £0 = £0.75. The seller received £0.75 for writing the option and it has zero value at expiration.


A buyer of puts or a seller of calls has short exposure to the underlying (will profit when the price of the underlying asset decreases). A buyer of calls or a seller of puts has long exposure to the underlying (will profit when the price of the underlying asset increases).


### LOS 69.c: Contrast forward commitments with contingent claims.

A **forward commitment** is a legally binding promise to perform some action in the future. Forward commitments include forward contracts, futures contracts, and most swaps.

A **contingent claim** is a claim (to a payoff) that depends on a particular event. Options are contingent claims; the event is the price of the underlying being above or below the exercise price. Credit default swaps are also considered contingent claims because the payment by the protection seller depends on a credit event occurring

<HR>

## Reading 69: Key Concepts

<HR>
    
### LOS 69.a

Forward contracts obligate one party to buy, and another to sell, a specific asset at a specific price at a specific time in the future.

Futures contracts are much like forward contracts, but are exchange-traded, liquid, and require daily settlement of any gains or losses.

A call option gives the holder the right, but not the obligation, to buy an asset at a specific price at some time in the future.

A put option gives the holder the right, but not the obligation, to sell an asset at a specific price at some time in the future.

In an interest rate swap, one party pays a fixed rate and the other party pays a floating rate, on a given amount of notional principal. Swaps are equivalent to a series of forward contracts based on a floating rate of interest.

A credit default swap is a contract in which the protection seller provides a payment if a specified credit event occurs.

### LOS 69.b

Call option value at expiration is Max(0, underlying price minus exercise price) and profit or loss is Max(0, underlying price minus exercise price) minus the option cost (premium paid).

Put value at expiration is Max(0, exercise price minus underlying price) and profit or loss is Max(0, exercise price minus underlying price) minus the option cost.

A call buyer (call seller) benefits from an increase (decrease) in the value of the underlying asset.

A put buyer (put seller) benefits from a decrease (increase) in the value of the underlying asset.

### LOS 69.c

A forward commitment is an obligation to buy or sell an asset or make a payment in the future. Forward contracts, futures contracts, and most swaps are forward commitments.

A contingent claim is a derivative that has a future payoff only if some future event takes place (e.g., asset price is greater than a specified price). Options and credit derivatives are contingent claims.



<hr>


## Quiz


<hr>

**Question 1**\
Interest rate swaps are:


**Answer**\
equivalent to a series of forward contracts.

**Explanation**\
A swap is an agreement to buy or sell an underlying asset periodically over the life of the swap contract. It is equivalent to a series of forward contracts. 
\
(Module 69.2, LOS 69.a)

<hr>

**Question 2**\
A call option is:


**Answer**\
the right to buy at a specific price.

**Explanation**\
A call gives the owner the right to call an asset away (buy it) from the seller. 
\
(Module 69.1, LOS 69.a)

<hr>

**Question 3**\
At expiration, the exercise value of a put option is:


**Answer**\
positive if the underlying asset price is less than the exercise price.

**Explanation**\
The exercise value of a put option is positive at expiration if the underlying asset price is less than the exercise price. Its exercise value is zero if the underlying asset price is greater than or equal to the exercise price. The exercise value of an option cannot be negative because the holder can allow it to expire unexercised. 
\
(Module 69.2, LOS 69.b)c)

<hr>

**Question 4**\
At expiration, the exercise value of a call option is the:

**Answer**\
greater of zero or the underlying asset price minus the exercise price.

**Explanation**\
If the underlying asset price is greater than the exercise price of a call option, the value of the option is equal to the difference. If the underlying asset price is less than the exercise price, a call option expires with a value of zero. 
\
(Module 69.2, LOS 69.b)

<hr>

**Question 5**
\
An investor writes a put option with an exercise price of £40 when the stock price is £42. The option premium is £1. At expiration the stock price is £37. The investor will realize a:


**Answer**\
loss of £2

**Explanation**\
Because the stock price at expiration is less than the exercise price, the buyer of the put option will exercise it against the writer. The writer will have to pay £40 for the stock and can only sell it for £37 in the market. However, the put writer collected the £1 premium for writing the option, which reduces the net loss to £2. 
\
(Module 69.2, LOS 69.b))

<hr>

**Question 6**\
Which of the following derivatives is a forward commitment?


**Answer**\
interest rate swap.

**Explanation**\
This type of custom contract is a forward commitment.
\
(Module 69.1, LOS 69.c)

<hr>

**Question 7**\
An investor who is short a put and an investor who takes a long forward position in the same asset for the same period will have the same profit at settlement when:


**Answer**\
$S_T – F_0(T) = p_0.$

**Explanation**\
The long forward will have a profit equal to the asset price at settlement minus the forward price. The put seller will have a profit of $p_0$ minus the payoff on the put, $\max (0, X – S_T)$. If the put expires out of the money, put seller makes $p_0$ and the forward buyer gets $S_T – F_0(T)$, so their profits will be equal when $S_T – F_0(T) = p_0$
\
(Module 69.1, LOS 69.c)

<hr>

**Question 8**\
A call option has an exercise price of £120, and the stock price is £105 at expiration. The expiration day value of the call option is:

**Answer**\
£0

**Explanation**\
A call option has an expiration day value of $\max(0, S − X)$. Here, $X$ is £120 and $S$ is £105. Because the call option is *out of the money* at expiration, its value is zero.
\
(Module 69.1, LOS 69.c)

<hr>

**Question 9**\
Jimmy Casteel pays a premium of £1.60 to buy a put option with an exercise price of £145. If the stock price at expiration is £128, Casteel's profit or loss from the options position is:

**Answer**\
£15.40

**Explanation**\
The put option will be exercised and has a value of £145-£128 = £17 $[\max(0, X − S)]$.  Therefore, Casteel receives £17 minus the £1.60
paid to buy the option. The profit is £15.40 ($£7 less £1.60).
\
(Module 69.2, LOS 69.b)

<hr>

**Question 10**\
Which of the following statements regarding call options is most accurate? The:

**Answer**\
breakeven point for the buyer is the exercise price plus the option premium.

**Explanation**\
The breakeven for the buyer and the seller is the exercise price plus the premium. The call holder will exercise if the market price exceeds the exercise price.
\
(Module 69.2, LOS 69.b)

<hr>

**Question 11**
\
A European call option on a stock has an exercise price of 42. On the expiration date, the stock price is 40. The value of the option at expiration is:


**Answer**\
zero.

**Explanation**\
For a call option, the value at expiration is zero if the price of the underlying is less than or equal to the exercise price. The holder will allow the option to expire unexercised.
\
(Module 69.2, LOS 69.b)

<hr>

**Question 12**
\
At expiration, the value of a call option is the greater of zero or the:

**Answer**\
underlying asset price minus the exercise price.

**Explanation**\
The value of a call option at expiration is its exercise value, which is $\max[0, S – X]$.
\
(Module 69.2, LOS 69.b)

<hr>

**Question 13**
\
Credit default swaps are *least accurately* characterized as:

**Answer**\
forward commitments.

**Explanation**\
Credit default swaps are contingent claims and not forward commitments because their payoff depends on a future event taking place. Credit derivatives are a form of insurance against a credit event.
\
(Module 69.1, LOS 69.a)
